le_utils/constants/file_formats.py

```diff
@@ -41,7 +41,7 @@
 
 # constants for HTML5 zip format
 HTML5 = "zip"
-HTML5_MIMETYPE = "application/zip"
+HTML5_MIMETYPE = ".zip"
```

Comment:
(I think there were some others that used this format as well, so we can revisit once we refactor the constants stuff)